packages/plugin-figma-blade-coverage/src/bladeLibraryConstants.ts

```diff
@@ -0,0 +1,8217 @@
+export const paymentLightThemeData = {
+  components: {
```

Comment:
Did we manually collected these IDs or do we have a script to generate? 

If manually, let's add a TODO to automate this, seems like a huge effort to maintain these IDs manually. (any additional colors, components, tokens could get missed)

Reply:
there's a script in the utils directory

Reply:
Oh noice!